bin/ci-product.sh

```diff
@@ -12,5 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-make test-images 
```

Comment:
maybe lest remove this from Makefile

Reply:
Apparently this is needed. We looked at that with @sopel39 and decided removing that would require understanding large part of this crap. So let's leave that.